tests/footprint2.py

```diff
@@ -455,9 +449,11 @@ def doTestPeaks(self, dwidth=0, dheight=0, x0=0, y0=0, threshold=10, callback=No
         #
         # Sort self.peaks in decreasing peak height to match Footprint.getPeaks()
         #
+        def peakKey(p):
+            return p[2] * -1.0
```

Comment:
Yes. That was grating on my mind as I iterated on solving the problems in this file...